app/assets/stylesheets/pages/watchlist.scss

```diff
@@ -0,0 +1,80 @@
+body.page-watchlist{
+  .ribbon-container {
+		position: relative;
+	}
+
+  .ribbon{
```

Comment:
Change this to `.corner-ribbon`, in future we might want to add a modifier such as `-tr` for top-right